sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -93,6 +98,29 @@ export class Constants {
     "workspaces/{workspaceId}/test-runs/{testRunId}/resulturi";
   public static readonly testResultsEndpoint: string =
     "workspaces/{workspaceId}/test-results/upload-batch";
+  public static readonly ERROR_MESSAGE: ErrorMessages = {
```

Comment:
NIT: These are not Error Messages, rather a mapping between status code -> api type -> error message

Reply:
sure